composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "pantheon/terminus-plugin-example",
+    "description": "An example Terminus command",
+    "type": "terminus-plugin",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": { "Pantheon\\TerminusHello": "src" }
+    },
+    "extra": {
+        "terminus": {
+            "compatible-version": "1.*"
```

Comment:
Maybe this should be `^1` or `>=1.0 <2.0.0`.  If someone wrote a plugin that required Terminus 1.1.0 or later, they might follow the above example and write `1.1.*`.  That would not be correct, as it would inadvertently exclude version 1.2.0 and later.  People are often confused by `^` and `~`, so perhaps between the two alternatives (`^1` or `>=1.0 <2.0.0`), the longhand version might be clearer.